files/familysearch-javascript-sdk/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "familysearch-javascript-sdk",
+  "repo": "FamilySearch/familysearch-javascript-sdk",
+  "files": {
+    "include": ["*.js"]
```

Comment:
I see, then please replace `*.js` with `familysearch-javascript-sdk.min.js`. No need for the unminified file in production